### DISCUSSION TITLE
FlowFields should not be filled

### DISCUSSION THREAD

**Discussion by ChristianHovenbitzer** [StefanMaron/BusinessCentral.LinterCop#142] [New Rule]
Currently there is no warning or indication that I'm about to fill a FlowField, which almost every time is not intendet. A rule to check the FieldClass on these occasions would be very appreciated.

The comment field on the Customer table is a FlowField.
![image](https://user-images.githubusercontent.com/91181840/153412047-597a010c-6a7d-4d04-969f-fa6da067ad2a.png)

**Comment by StefanMaron** [comment]
Only if the flow field is set to `Editable = false`?  
see https://github.com/StefanMaron/BusinessCentral.LinterCop/wiki/LC0001

**Reply by ChristianHovenbitzer** [reply]
As this is already covered by another rule I would suggest to specify the reasoning by a comment in any case.
Filling a FlowField is not common and a comment explaning why this FlowField is filled is very helpful.
